modules/manually-rotating-cloud-creds-vsphere.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
```

Comment:
This file is not actually used but I want to preserve it for the moment until the PR is reviewed. If nothing is needed from it I will remove in a later commit :)